.github/workflows/build-publish-workflow.yml

```diff
@@ -29,11 +32,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Set up JDK 8
+      - name: Set up JDK ${{ inputs.java-version }}
```

Comment:
>This change will create a cascading impact on the repositories that are currently dependent on it.

Wouldn't there be a way to default to `8` is the version isn't provided explicitly?

Reply:
Yeah there's one, not sure, I haven't used before. Something like;
```yml
with:
    distribution: 'temurin'
     java-version: ${{ inputs.java-version || '8' }}
```